hrrecruitment/CRM/HRRecruitment/Form/Application.php

```diff
@@ -238,7 +238,7 @@ public function postProcess() {
       }
 
       //process Custom data
-      CRM_Core_BAO_CustomValueTable::postprocess($params,CRM_Core_DAO::$_nullArray, 'civicrm_case', $caseObj->id, 'Case');
+      CRM_Core_BAO_CustomValueTable::postprocess($params, 'civicrm_case', $caseObj->id, 'Case');
```

Comment:
Would be good to replace 'postprocess' with 'postProcess'. PHP method names are case-insensitive, but as they say (here: http://fr.php.net/manual/en/functions.user-defined.php) :
"Note: Function names are case-insensitive, though it is usually good form to call functions as they appear in their declaration."